drivers/docker/test-resources/docker/gen_stopsignal.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+
+# Create the tarball used in TestDockerDriver_StopSignal
+echo "FROM busybox:1.29.3\nSTOPSIGNAL 19" > tmpfile && docker build -t busybox:1.29.3-stopsignal - < tmpfile && rm tmpfile
```

Comment:
I suspect you can avoid needing the tmp file. Also it might a bit nicer to use here-doc style in this case to avoid new lines:

```suggestion
cat <<'EOF' | docker build -t busybox:1.29.3-stopsignal -
FROM busybox:1.29.3
STOPSIGNAL 19
EOF
```